src/org/zaproxy/zap/extension/jxbrowser/ZapAddOn.xml

```diff
@@ -7,7 +7,7 @@
 	<url></url>
 	<changes>
 	<![CDATA[
-		Update the help to mention the bundled JxBrowser version.<br>
```

Comment:
This change could  be kept. The help should be updated to mention the new version.

Reply:
done